lib/suppressions.cpp

```diff
@@ -112,6 +112,92 @@ std::string Suppressions::parseXmlFile(const char *filename)
     return "";
 }
 
+std::vector<Suppressions::Suppression> Suppressions::parseMultiSuppressComment(std::string comment, std::string *errorMessage)
+{
+    std::vector<Suppression> suppressions;
+    if (comment.size() < 2)
+        return suppressions;
+
+    size_t suppress_position = comment.find("cppcheck-suppress");
```

Comment:
please use `const` for local variables when possible

please use the c++ type `std::size_t`. It is not standard behavior that C headers are included in the C++ headers.